### DISCUSSION TITLE
Add metadata to file

### DISCUSSION THREAD

**Discussion by vincentarius** [supabase/supabase#20805] [Feature Requests]
I would like to add metadata to a file that I store in storage. For example like metadata that can be added to a user

**Comment by GaryAustin1** [comment]
https://github.com/orgs/supabase/discussions/5479

**Comment by bombillazo** [comment]
@vincentarius  I've created an issue [here](https://github.com/supabase/storage/issues/439) to follow up, would be helpful to get some traction

**Comment by vincentarius** [comment]
Great! Just tell me what I can do to help.Verstuurd vanaf mijn iPhoneOp 15 feb 2024 om 03:42 heeft Hector Ayala ***@***.***> het volgende geschreven:﻿
@vincentarius  I've created an issue here to follow up, would be helpful to get some traction
